packages/esm-commons-lib/src/dashboards/createDashboard.tsx

```diff
@@ -5,22 +5,12 @@ import { DashboardLinkConfig } from '../types';
 import { DashboardExtension } from './DashboardExtension';
 import { BrowserRouter } from 'react-router-dom';
 
-export const createDashboardLinkWithCustomTitle = (db: DashboardLinkConfig) => {
```

Comment:
we don't need this anymore with the introduction of the `path` and `title` as well us custom meta in the `routes.json`